addons/point_of_sale/static/src/js/screens.js

```diff
@@ -1111,7 +1131,9 @@ var ProductScreenWidget = ScreenWidget.extend({
                 }
             }
         }
-    },
+        this.buffered_keys = []
+    }
```

Comment:
ends with comma (,) to avoid modify that line in case someone wants to add stuff